### DISCUSSION TITLE
Deformed website when hosted github pages

### DISCUSSION THREAD

**Discussion by zakiayoubi** [community/community#125029] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hey guys, I need some help.
My website in development looks really good in the browser, but when I hosted it on github pages. it's kinda broken and deformed why? some of the images are out of shape and some of the colors aren't there, but the rest of it looks good.

**Comment by LiteBrite82** [comment]
Hi @zakiayoubi 

Can you share a link to the repo and deployment?
